src/metldata/builtin_transformations/common/assumptions/source_assumptions.py

```diff
@@ -17,6 +17,9 @@
 
 from schemapack.spec.schemapack import SchemaPack
 
+from metldata.builtin_transformations.add_content_properties.path import (
+    resolve_schema_object_path,
+)
```

Comment:
Didn't check before, but this (and `resolve_data_object_path`) is something for the refactoring list.
Common probably shouldn't pull in dependencies from any transformation and this should be defined in common instead.

Reply:
I didn't know that it was on a refactoring list. 

It needs to be moved to the common though. I left it to clean-up.